pkg/webui/console/store/middleware/logics/user.js

```diff
@@ -15,14 +15,27 @@
 import * as user from '../../actions/user'
 import api from '../../../api'
 import * as accessToken from '../../../lib/access-token'
+import { isUnauthenticatedError } from '../../../../lib/errors/utils'
 import createRequestLogic from './lib'
 
 export default [
   createRequestLogic({
     type: user.LOGOUT,
     async process() {
       try {
+        // Logout of the Console first
         await api.console.logout()
+
+        try {
+          // Then logout of the OAuth Provider
+          await api.oauth.logout()
```

Comment:
I dont think we should trigger this from the console

Reply:
What do you suggest then?